control/veda_file_ingest.py

```diff
@@ -243,10 +184,8 @@ def database_record(self):
             self.full_filename += "." + self.video_proto.file_extension
 
         if not os.path.exists(self.full_filename):
-            ErrorObject().print_error(
-                message='Ingest: File Not Found'
-            )
-            return None
+            LOGGER.exception('[VIDEO_INGEST] File Not Found {veda_id}', veda_id=self.video_proto.veda_id)
```

Comment:
I think this will not work. this should be 

```python
LOGGER.exception('[VIDEO_INGEST] File Not Found %s', self.video_proto.veda_id)
```

Reply:
Right, sorry, should be .format or string formatting. Fixed.